examples/example_morph.m

```diff
@@ -28,7 +28,7 @@
 % Alternative way to add 3D point to main using dynamic top-level function.
 main.push();
 n.pt3dadd(201, 2, 3, 1);
-main.pop();
+neuron.stack.pop_sections(1);
```

Comment:
For an example, instead of using something internal to the toolbox itself, let's use NEURON's built-in:
```
n.pop_section()
```
(but please make sure it works :) )

Reply:
It decreases `refcount` by 1, so it looks like that also works.